lazyfs/include/lazyfs/lazyfs.hpp

```diff
@@ -144,12 +177,13 @@ class LazyFS : public Fusepp::Fuse<LazyFS> {
      * @param config LazyFS configuration
      * @param faults_handler_thread A thread to handle fault requests
      * @param fht_worker The faults worker logic method
+
```

Comment:
```suggestion
```

Reply:
I'll update this section based on your suggestion